services/minespace-web/src/components/pages/Project/ProjectSummaryPage.tsx

```diff
@@ -71,7 +75,11 @@ export const ProjectSummaryPage = () => {
     : mine?.mine_guid === mineGuid;
   const [isLoaded, setIsLoaded] = useState(isDefaultLoaded);
   const [isEditMode, setIsEditMode] = useState(isDefaultEditMode);
-  const projectFormTabs = null;
+  const projectFormTabs = getProjectFormTabs(
+    amsFeatureEnabled,
+    isCore,
```

Comment:
Y'know, I even noticed that this function was throwing errors! 
Technically this should _always_ be false as this is a MineSpace file, but I'm not bothered by a bit of extra robustness.

Reply:
Yeah, I thought might as well add the actual flag for that imaginary case where this component gets used in Core at some point for some weird unknown reason.